Ventivader/Ventivader/Utilities/LoadingOverlay.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  LoadingOverlay.swift
+//  Ventivader
+//
+//  Created by Al on 19/04/20.
+//  Copyright © 2020 calculator. All rights reserved.
+//
+import UIKit
+import Foundation
+import SwiftGifOrigin
+
+
+public class LoadingOverlay {
+    
+    var overlayView = UIView()
+    //var activityIndicator = UIActivityIndicatorView()
+    
+    private lazy var bleLoaderView: UIView = {
+        let containerView = UIView()
+        containerView.backgroundColor = .white
+        
+        let animationView = UIImageView()
+        animationView.loadGif(name: "bleConnectingAnimation")
+        
+        let titleLabel = UILabel()
+        titleLabel.text = NSLocalizedString("Connecting BLE...", comment: "title for connecting BLE")
+        titleLabel.updateFont(name: VentivaderFonts.titleFont)
+        titleLabel.textAlignment =  .center
+        
+        containerView.addSubview(animationView)
+        containerView.addSubview(titleLabel)
+        
+        animationView.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        
+        // Animation View constraints
```

Comment:
later we can refactor this autolayout code into a method.

Reply:
I did this refactor